docs/userguide/configuration.rst

```diff
@@ -1104,6 +1104,26 @@ Default: 120.0 seconds.
 Socket timeout for reading/writing operations to the Redis server
 in seconds (int/float), used by the redis result backend.
 
+.. setting:: redis_retry_on_timeout
+
+``redis_retry_on_timeout``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Default: :const:`False`
+
+To retry reading/writing operations on TimeoutError to the Redis server,
+used by the redis result backend.
+
+.. setting:: socket_keepalive
```

Comment:
@Seleznev-nvkz per line 188 in `celery/backends/redis.py` shouldn't this be `redis_socket_keepalive` rather than `socket_keepalive`?

Reply:
Great catch! Thank you.

Reply:
Done https://github.com/celery/celery/pull/6000